### PR TITLE
allow space after attribute name, associated spec

### PR DESCRIPTION
**Pull request by zoo1**
### Description of the Change

Added space after attributes, to fix bug #81 

### Benefits

Improved XML grammer

### CODE REVIEW

File: grammars/xml.cson [hunk 1]
```diff
@@ -387,7 +387,7 @@
             'name': 'punctuation.separator.namespace.xml'
           '4':
             'name': 'entity.other.attribute-name.localname.xml'
-        'match': '(?:^|\\s+)(?:([-\\w.]+)((:)))?([-\\w.:]+)='
+        'match': '(?:^|\\s+)(?:([-\\w.]+)((:)))?([-\\w.:]+) *='
```
Comment (50Wliu) [inline grammars/xml.cson]: Can ` *` be replaced with `\\s*` for clarity?

**Comment by zoo1** [comment]
xml

**Comment by zoo1** [comment]
@50Wliu updated!

**Comment by jens1o** [comment]
Thanks!

**Pull request merged**